src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "ball number " + number + ", color " + color;
+    }

```

Comment:
I'd recommend generating toString method using Intellij IDEA tools to make this method more clear and consistent